src/screens/Settings/index.tsx

```diff
@@ -451,14 +465,19 @@ function Settings() {
           )}
           <span className="save">{t('info.settings')}</span>
           <button
-            className="button is-text"
-            onClick={() =>
+            className={`button is-text ${
```

Comment:
We have a pretty useful npm package to deal with this.
You can just `import classNamesfrom 'classnames'`
 and the use:
```
className={classNames('button', 'is-text', {
              isSuccess: isCopiedToClipboard
            })}
```
also, avoid using class names like that with `-`, react might go crazy with that sometimes. 
I added a reminder to myself to rename all classes that have a dash to be camel case.